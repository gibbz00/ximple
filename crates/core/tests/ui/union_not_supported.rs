#[derive(ximple::ToXml)]
union Foo {
    foo: usize,
}

fn main() {}
