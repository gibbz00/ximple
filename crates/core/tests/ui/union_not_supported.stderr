error: unions are not supported
 --> tests/ui/union_not_supported.rs:2:1
  |
2 | union Foo {
  | ^^^^^
